webui/apps/docs/src/content/docs/docs/howto/llm_authentication.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Using LLM APIs
+title: LLM Authentication
```

Comment:
We proposed changing the "How To" section titles, e.g., "Authenticating with LLMs"

Reply:
@flynntsang , I assumed you meant the sidebar

so in the sidebar it would read
How to..
* Authenticate your LLM

but then the title would remain `LLM Authentication`

let me know what you think.

Reply:
Yes on the sidebar. Personally, I prefer "How to authenticate your LLM" in the doc title, as this page is giving step-by-step instructions on how to do something. It's not just a reference page.